client/components/Layout/Header.tsx

```diff
@@ -43,6 +45,14 @@ const Header = ({
     setProfileMenuAnchor(undefined);
   };
 
+  const logOut = async (): Promise<void> => {
+    await API.put(`/users/logout`).catch((e) => {
```

Comment:
use try/catch if you're using `await`
(+ are you sure you want to console.error ?)